stock_picking_auto_create_lot/models/stock_move_line.py

```diff
@@ -35,5 +35,4 @@ def set_lot_auto(self):
         for line in self:
             lot = first(lots_by_product[line.product_id.id])
             line.lot_id = lot
-            if lot.product_id.tracking == "serial":
-                lots_by_product[line.product_id.id] -= lot
+            lots_by_product[line.product_id.id] -= lot
```

Comment:
@aayartsev Wasn't this case covered by a test ?

Reply:
Thanks for feedback
I opened new PR #1566 )